pyproject.toml

```diff
@@ -38,6 +38,7 @@ dependencies = [
   "networkx",
   "lxml",
   "PyGithub",
+  "mypy",
```

Comment:
Do we have to add mypy here? It's already in the current pyproject.toml file. In type_check.yaml, we should be able to install it via `pip install ".[all]"`?

Reply:
you are right. I was not aware. Thanks